src/Compatibility/Microsoft.DotNet.PackageValidation/ApiCompatRunner.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    /// <summary>
+    /// Runs ApiCompat over different assembly tuples.
+    /// </summary>
+    public class ApiCompatRunner
+    {
+        private static List<(Stream leftAssemblyStream, Stream rightAssemblyStream, string assemblyName, string source, string errorMessage)> queue = new();
```

Comment:
Do we really want this data as `static`? What happens if the Task is called twice? Or twice at the same time?